Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -251,9 +251,10 @@ static void BPWritten(const BPCmd& bp)
 					height = MAX_XFB_HEIGHT;
 				}
 
-				u32 width = bpmem.copyMipMapStrideChannels << 4;
-
-				Renderer::RenderToXFB(destAddr, srcRect, width, height, s_gammaLUT[PE_copy.gamma]);
+				u32 stride = bpmem.copyMipMapStrideChannels << 4;
+				WARN_LOG(VIDEO, "RenderToXFB: destAddr: %08x | srcRect {%d %d %d %d} | fbWidth: %u | fbStride: %u | fbHeight: %u",
+					destAddr, srcRect.left, srcRect.top, srcRect.right, srcRect.bottom, bpmem.copyTexSrcWH.x + 1, stride, height);
+				Renderer::RenderToXFB(destAddr, srcRect, bpmem.copyTexSrcWH.x + 1, stride, height, s_gammaLUT[PE_copy.gamma]);
```

Comment:
I'm mostly against passing redundant information. The width is now passed twice.